route_following_plugin/config/parameters.yaml

```diff
@@ -21,4 +21,4 @@ lane_following_plugin : "inlanecruising_plugin"
 
 # Double : The minimum length a maneuver can be when it is being modified to support stopping behavior.
 # Units : meters
-min_maneuver_length : 45.0 # TODO update in lane cruising so that such an exterme value is not needed
```

Comment:
This also confirmed to have not been used and not needed for our current map. I think this is because the length is long and we don't actually plan stop_and_wait anymore